.github/workflows/ci.yml

```diff
@@ -21,10 +21,32 @@ jobs:
         - php: 8.1
           db: pgsql
           phpunit: true
-          composer_args: --prefer-lowest
+          composer_args: --prefer-lowest --prefer-source
```

Comment:
You don't need `--prefer-source` on any of these, the generated composer.json file will prefer source for silverstripe/* - https://github.com/silverstripe/silverstripe-postgresql/actions/runs/4160191510/jobs/7196950776#step:9:368